<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="twMain">
      <column>
       <property name="text">
        <string>db</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ver</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>287</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_Databases">
    <property name="title">
     <string>&amp;Databases</string>
    </property>
    <addaction name="actionDbCreate"/>
    <addaction name="actionDbRename"/>
    <addaction name="actionDbDelete"/>
    <addaction name="separator"/>
    <addaction name="actionDbInit"/>
    <addaction name="actionDbUpgrade"/>
    <addaction name="actionDbDeinit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menu_Service">
    <property name="title">
     <string>&amp;Service</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Databases"/>
   <addaction name="menu_Service"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDbCreate"/>
   <addaction name="actionDbDelete"/>
   <addaction name="actionDbInit"/>
   <addaction name="actionDbDeinit"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/exit.png</normaloff>:/icon/icons/16x16/exit.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/info.png</normaloff>:/icon/icons/16x16/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/qt4.png</normaloff>:/icon/icons/16x16/qt4.png</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/help.png</normaloff>:/icon/icons/16x16/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="actionDbCreate">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/db_add.png</normaloff>:/icon/icons/16x16/db_add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create DB</string>
   </property>
  </action>
  <action name="actionDbDelete">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/db_remove.png</normaloff>:/icon/icons/16x16/db_remove.png</iconset>
   </property>
   <property name="text">
    <string>De&amp;lete DB</string>
   </property>
  </action>
  <action name="actionDbRename">
   <property name="text">
    <string>Re&amp;name DB</string>
   </property>
  </action>
  <action name="actionDbInit">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/db_commit.png</normaloff>:/icon/icons/16x16/db_commit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Init DB</string>
   </property>
  </action>
  <action name="actionDbDeinit">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/db_update.png</normaloff>:/icon/icons/16x16/db_update.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Deinit DB</string>
   </property>
  </action>
  <action name="actionDbUpgrade">
   <property name="text">
    <string>Upgrade DB</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../qtselta.qrc">
     <normaloff>:/icon/icons/16x16/configure.png</normaloff>:/icon/icons/16x16/configure.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../qtselta.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
